be/src/olap/rowset/segment_v2/segment_iterator.cpp

```diff
@@ -892,12 +892,40 @@ void SegmentIterator::_read_columns_by_rowids(std::vector<ColumnId>& read_column
                                               vectorized::MutableColumns* mutable_columns) {
     SCOPED_RAW_TIMER(&_opts.stats->lazy_read_ns);
     size_t start_idx = 0;
+    size_t batch_size = 256;
```

Comment:
Maybe here can directly use `binary search` to find the boundary instead of find by `batch_size`?

Reply:
OK，I'll give it a test.